lib/media/track/remotemediatrack.js

```diff
@@ -32,6 +34,22 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
           writable: true
         }
       });
+
+      deprecateEvents('RemoteMediaTrack', this, new Map([
+        ['unsubscribed', null]
+      ]), this._log);
+    }
+
+    /**
+     * The {@link RemoteMediaTrack}'s ID.
+     * @property {Track.ID}
+     * @deprecated
```

Comment:
I recommend we suggest using Track `name` or Track `sid` instead. **EDIT:** We do have the mention of `.sid` below; we can add `.name` and then include text in the JSDoc `@deprecated` annotation.